src/test/java/com/easypost/EventTest.java

```diff
@@ -53,10 +62,16 @@ public void testAll() throws EasyPostException {
      */
     @Test
     public void testRetrieve() throws EasyPostException {
-        Event retrievedEvent = Event.retrieve(globalEvents.getEvents().get(0).getId());
+        _vcr.setUpTest("retrieve");
+
+        EventCollection eventCollection = getBasicEventCollection();
+        List<Event> events = eventCollection.getEvents();
+        Event event = events.get(0);
+
+        Event retrievedEvent = Event.retrieve(event.getId());
 
-        assertTrue(retrievedEvent instanceof Event);
-        assertTrue(retrievedEvent.getId().startsWith("evt_"));
-        assertEquals(globalEvents.getEvents().get(0).getId(), retrievedEvent.getId());
+        assertInstanceOf(Event.class, retrievedEvent);
+        // Must compare IDs because other elements of objects may be different
```

Comment:
I'm unsure that this is true of events, can we double check?

Reply:
Cannot currently do whole-object comparisons in Java.

Reply:
Hmmm, why not? I thought we had a PR dedicated to just overcoming this problem.